### PR TITLE
Platoniq web plugins

### PR DESCRIPTION
**Pull request by LauraParejaNazaries**
New image plugin to be able to add an image with a decorator in a project

### CODE REVIEW

**Comment by LauraParejaNazaries** [comment]
Small changes done to pass Rubocop rules

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "workbench.colorCustomizations": {
+        "activityBar.background": "#063337",
+        "titleBar.activeBackground": "#08474D",
+        "titleBar.activeForeground": "#F0FCFD"
+    }
+}
```
Comment (davidbeig) [inline .vscode/settings.json]: Could you remove the .vscode folder and add it to the .gitignore?

**Review by davidbeig — CHANGES_REQUESTED** [review]
Sorry for the delay with this review @LauraParejaNazaries!

I'm only asking for the removable of the .vscode folder, since it might interact with each other configurations. Also, it's weird that this plugin, that its meant to be for the journal can have tags that are not defined in [journal theme](https://github.com/Platoniq/jekyll-theme-platoniq-journal)  but are in the [platoniq.net theme](https://github.com/Platoniq/jekyll-theme-platoniq.net).

Thanks for this PR!

**Comment by LauraParejaNazaries** [comment]
I just made the change. Regarding using the same repo. We did not have any info about what plugins were or how they work, nor did we get a response about what they were when we asked. But doing our own investigation we realized many of the implementations they were asking us for were already made in this repo. To not repeat code that is shared among projects we though it was a good idea to include it here. When we commented on it no one told us we should not do that.

**Pull request merged**